modules/tinymce/CHANGELOG.md

```diff
@@ -1397,7 +1397,7 @@ Full documentation for the version 5 features and changes is available at https:
 
 Developer preview 1
 
-Initial list of features and changes is available at https://tiny.cloud/docs-preview/release-notes/new-features/
+Initial list of features and changes is available at https://tiny.cloud/tinymce/5/release-notes/release-notes50.html
```

Comment:
```suggestion
Initial list of features and changes is available at https://www.tiny.cloud/docs/tinymce/5/release-notes/release-notes50.html
```
This link is also incorrect (missing the `docs` part of the path).

Reply:
Lee walked me through this, so accepting on Mike's behalf due to time constraints.